src/sentry/static/sentry/app/views/admin/adminBuffer.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+
+import InternalStatChart from 'app/components/internalStatChart';
+
+const AdminBuffer = () => {
+  const since = new Date().getTime() / 1000 - 3600 * 24 * 7;
```

Comment:
Do we need to floor/round this? It will probably be something like `1604561186.888`

Reply:
It probably doesn't matter. I just left it what it was.